### DISCUSSION TITLE
About the bug of tool sql-schema-to-typescript-interface

### DISCUSSION THREAD

**Discussion by Robin-ou** [he3-app/he3-feedback#148] [Bug Report]
### Version
sql-schema-to-typescript-interface 1.0.0

### Environment
Windows v1.3.2

### Steps to reproduce

table fieldName is UnderScoreCase，Like "product_name",After paraphrasing,should be like Camel-Case "ProductName"

**Comment by lyzhang1999** [comment]
Thanks for your report, we have created an issues to track please check: https://github.com/he3-app/he3-feedback/issues/149
